elixir/apps/api/lib/api/controllers/controller_helpers.ex

```diff
@@ -0,0 +1,23 @@
+defmodule API.ControllerHelpers do
```

Comment:
```suggestion
defmodule API.Pagination do
```

and then it would be better to jus use `Pagination.params_to_list_opts(params)` everywhere instead of doing `import API.ControllerHelpers`. 

`import` adds compile-time dependency and pollutes the namespace of the module, plus it's not super clear where the function is coming from where the module has a bunch of `import` or `use`'s.